locales/nl/nl.json

```diff
@@ -485,25 +485,25 @@
     "Saint Barthelemy": "St. Barthélemy",
     "Saint Barthélemy": "Saint Barthélemy",
     "Saint Helena": "Sint-Helena",
-    "Saint Kitts and Nevis": "Saint Kitts and Nevis",
+    "Saint Kitts and Nevis": "Saint Kitts en Nevis",
     "Saint Kitts And Nevis": "St. Kitts En Nevis",
     "Saint Lucia": "Saint Lucia",
     "Saint Martin": "St. Martin",
     "Saint Martin (French part)": "Saint Martin (French part)",
     "Saint Pierre and Miquelon": "Saint Pierre and Miquelon",
-    "Saint Pierre And Miquelon": "Min. orde: 1set \/ Sets",
+    "Saint Pierre And Miquelon": "Saint Pierre And Miquelon",
```

Comment:
you don't translate "and" ?

Reply:
Should be solved in https://github.com/Laravel-Lang/lang/pull/1733/commits/26f0926779cbdd1cac78503d99940599f81c7178